GOVERNANCE.md

```diff
@@ -128,10 +135,71 @@ the issue tracker is:
   either the proposal is dropped or the objecting members are persuaded. If
   there is an extended impasse, a motion for a vote may be made.
 
+## Collaborator Nominations
+
+Any existing Collaborator can nominate an individual making significant
+and valuable contributions across the Node.js organization to become a new
+Collaborator.
+
+To nominate a new collaborator, open an issue in the [nodejs/node][]
+repository, with a summary of the nominee's contributions, including but
+not limited to:
```

Comment:
The language "including but not limited to" suggests that all of the below bullet items should always be included.  I'm OK with that, but just leaving this comment to make sure that is 100% the intention here.

Reply:
@Trott Yes it does look like we are requiring people to list all of the items, which is not really what I meant... I'll change it to `For example:`